README.md

```diff
@@ -12,6 +12,8 @@ are also very welcome.
 
 Install the normal way: `npm install json-schema-to-markdown`
 
+or if using cli, install globally: `npm install -g json-schema-to-markdwon`
```

Comment:
typo in package name

Reply:
Fixed https://github.com/saibotsivad/json-schema-to-markdown/pull/13/commits/6a4033dca9f602fe31f7f8c1465e3043850a6e26